packages/wow-ui/src/components/Chip/index.tsx

```diff
@@ -107,11 +107,11 @@ const Chip: ChipComponent & { displayName?: string } = forwardRef(
 
     return (
       <Component
-        ischecked
         aria-checked={clickable ? ischecked : undefined}
         aria-disabled={disabled}
         aria-label={`chip ${ischecked ? "activated" : "inactivated"}`}
         data-selected={ischecked}
+        ischecked="true"
```

Comment:
p3;
ischecked 는 불리언 값에 따라 달라지는 부분 맞을까요?? 형변환만 해주면 될 거 같아서용 @eugene028 님께서 확인 한번 해주심 감사하겠습니다~~
```suggestion
        ischecked={String(ischecked)}
```

Reply:
원래 `ischecked`로 바로 전달되는 체크여부 불리언값이라 말씀하신게 맞는 것 같아요~!